pytensor/tensor/math.py

```diff
@@ -1945,6 +1945,18 @@ def dot(l, r):
     """Return a symbolic dot product.
 
     This is designed to work with both sparse and dense tensors types.
+
+    Example usage with PyMC:
+
+    .. code:: python
+
+        import pymc as pm
+        import pytensor.tensor as pt
+
+        with pm.Model() as model:
+            x = pm.Normal('x', mu=0, sd=1, shape=2)
+            y = pm.Normal('y', mu=0, sd=1, shape=2)
+            z = pt.dot(x, y)
```

Comment:
Don't show pytensor?

```suggestion
            z = pt.math.dot(x, y)
```

Reply:
See comment above.